conflux/caip10.md

```diff
@@ -0,0 +1,121 @@
+---
+namespace-identifier: conflux-caip10
+title: Conflux Namespace - Addresses
+author: iosh(@iosh)
+status: Draft
+type: Standard
+created: 2024-08-09
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+_For context, see the [CAIP-10][] specification._
+
+## Rationale
+
+The Conflux Network EOA address is a base32-encoded address.
+
+These are derived directly from the hexadecimal-encoded public keys corresponding to the private keys controlling them, but calculated over a network identifier, address type, and checksum value, address including a distinctive network prefix ("cfx" "cfxtest" or "net[networkId]").
+
+The Conflux core space hex address is a concatenation of a 4-bit type indicator and the rightmost 156-bit Keccak digest of the associated public key of the private key.
+
+There are currently three types of indicators:
+
+(0x)1: Represents the address of an EOA account.
+(0x)8: Represents the address of a contract
+(0x)0: Represents the address of an internal contract,
+
+Constructing an address:
+
+```
+encode(0x1a2f80341409639ea6a35bbcab8299066109aa55, "cfx") // cfx:aarc9abycue0hhzgyrr53m6cxedgccrmmyybjgh4xg
+```
+
+1. Network Prefix: cfx(mainnet network ID 1029), cfxtest(testnet network ID 1), net + network ID (ID is not 1 or 1029)
```

Comment:
i'm confused by the `not `1 or 1029` part? do you use the integer chainId or the string form (`cfx`, etc)?